vizro-core/examples/default/app.py

```diff
@@ -526,9 +526,25 @@ def create_home_page():
     ],
     navigation=vm.Navigation(
         pages={
-            "Analysis": ["Homepage", "Variable Analysis", "Relationship Analysis", "Country Analysis"],
-            "Summary": ["Continent Summary"],
-        }
+            "Analysis": ["Homepage", "Variable Analysis"],
+            "Summary": ["Relationship Analysis", "Continent Summary", "Country Analysis"],
+        },
+        selector=vm.NavBar(
+            items=[
+                vm.NavItem(
+                    pages={
+                        "Analysis": ["Homepage", "Variable Analysis"],
+                        "Summary": ["Relationship Analysis", "Continent Summary"],
+                    },
+                    icon="",
+                    text="Features",
+                ),
+                vm.NavItem(
+                    pages=["Country Analysis"],
+                    text="Country Analysis is the set text for this icon",
+                ),
+            ]
+        ),
     ),
```

Comment:
Correct, Navigation.pages don't have to be provided and in this case will be ignored as selector takes precedence. 

I will add examples in the PR description to show possible configurations and the outcomes. 

I left the navigation.pages in to make it easier to test behaviour and outcomes of navigation.pages and navigation.selector, but it would be reverted for merge. But you are right, it is confusing.

Reply:
Perfect, thanks - that would be helpful!